### PR TITLE
Test Fixes

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

This should get the currently failing tests on Linux boxes passing again. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ for (lbl in labels) {
                 condaEnv("isis3") {
                     // Environment
                     loginShell """
-                        conda install -c conda-forge python=3
+                        conda install -c conda-forge python=3 findutils
```
Comment (Kelvinrr) [inline Jenkinsfile:37]: temporary fix, PR into the docker repo incoming. Not sure how that process will go to get the new containers running (had flakiness from updating image:latest and not being pulled as k8s doesn't see it as a newer version of the container ) so I don't want to count on it too much at the moment.

File: isis/src/base/objs/EmbreeShapeModel/EmbreeShapeModel.truth [hunk 2]
```diff
@@ -46,7 +46,7 @@ Embree shape model status:
 Model has intersection?  true
   Surface Point: ( 0.2908 ,  -0.0019 ,  0.0005 )
 Model has normal?  true
-  Surface Normal: ( 0.847198 ,  -0.194495 ,  0.494395 )
+  Surface Normal: ( 0.8471 ,  -0.1944 ,  0.4943 )
```
Comment (acpaquette) [inline isis/src/base/objs/EmbreeShapeModel/EmbreeShapeModel.truth:49]: Is this rounding a Mac vs Linux issue?

Comment (jessemapel) [reply isis/src/base/objs/EmbreeShapeModel/EmbreeShapeModel.truth:49]: Sort of, Embree only does floating point precision calculations and we sometimes see very small differences based on OS.

**Review by jessemapel — APPROVED** [review]
Hopefully this makes the test less flaky

**Review by krlberry — APPROVED** [review]

**Pull request merged**